samples/tfm_integration/tfm_secure_partition/dummy_partition/tfm_manifest_list.yaml.in

```diff
@@ -17,7 +17,6 @@
       "manifest": "${APPLICATION_SOURCE_DIR}/dummy_partition/tfm_dummy_partition.yaml",
       "output_path": "${TFM_BINARY_DIR}/dummy_partition",
       "tfm_partition_ipc": true,
-      "conditional": "TFM_PARTITION_DUMMY_PARTITION",
```

Comment:
This commit is already merged to main, should rebase and get rid of it.

Reply:
done